src/components/overlays/TheUpdateTacticOverlay.vue

```diff
@@ -0,0 +1,142 @@
+                v-model="selectedTactic.map"
+                color="primary"
+                hide-no-data
+                hide-selected
+                :label="$t('tactic.overlay.maps')"
+                :placeholder="$t('tactic.overlay.search')"
+                prepend-icon="fa-search"
+                autocomplete="new-password"
+                return-object
+              >
+                <template v-slot:item="data">
+                  <v-list-item-avatar size="29" class="custom-list-item-avatar">
+                    <img :src="data.item.icon" :alt="data.item.name">
+                  </v-list-item-avatar>
+                  <v-list-item-content class="custom-list-item-content">
+                    <v-list-item-title v-text="data.item.name"></v-list-item-title>
+                  </v-list-item-content>
+                </template>
+              </v-autocomplete>
+            </v-card-actions>
+            <v-spacer></v-spacer>
+            <v-card-subtitle>
+              <v-btn
+                color="primary"
+                @click="updateTacticOnClickHandler()"
+              >
+                {{ $t('tactic.updateTacticOverlay.update') }}
+              </v-btn>
+            </v-card-subtitle>
+          </v-col>
+          <v-divider class="d-none" vertical />
+          <v-col>
+            <v-card-subtitle>
+              <v-img v-if="selectedTactic.map.icon" :src="selectedTactic.map.icon" max-width="200px"/>
+            </v-card-subtitle>
+            <v-card-subtitle>{{ selectedTactic.map.name }}</v-card-subtitle>
+          </v-col>
+        </v-row>
+      </v-card>
+    </v-dialog>
+</template>
+<script lang="ts">
+import Component, { mixins } from 'vue-class-component'
+import { EventBus } from '@/event-bus'
+import { RoomGetters } from '@/store/modules/room'
+import { Action, Getter } from 'vuex-class'
+import { Api, Tactic } from '@/store/modules/types'
+import { StageActions } from '@/store/modules/stage'
+import { CustomStageConfig } from '@/util/PointerEventMapper'
+import { CanvasAction } from '@/store/modules/canvas'
+import { CanvasElement, CanvasElementHistory } from '@/types/Canvas'
+import TacticWatcher from '@/mixins/TacticWatcher'
+
+@Component({
+  name: 'TheUpdateTacticOverlay',
+  mixins: [TacticWatcher]
+})
+export default class TheUpdateTacticOverlay extends mixins(TacticWatcher) {
+  @Getter(`room/${RoomGetters.GAME_API}`) gameApi!: Api[]
+  @Action(`stage/${StageActions.SET_CONFIG}`) setConfig!: (config: CustomStageConfig) => void
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT}`) setCanvasElements!: (canvasElements: CanvasElement[]) => void
+  @Action(`canvas/${CanvasAction.SET_CANVAS_ELEMENT_HISTORY}`) setCanvasElementsHistory!: (canvasElements: CanvasElementHistory[]) => void
+
+  search = ''
+  selectedTactic: Tactic | {} = {}
+  overlay = false
+  created () {
+    EventBus.$on('openManageTacticsOverlay', (tactic: Tactic) => {
+      this.overlay = !this.overlay
```

Comment:
since this is only triggered on open, set the overlay to true instead of toggling it

Reply:
good point will fix